tools/kubernetes-client.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+set -eu
+set -x
+
+if [[ "${CLIENT:-}" = "yes" ]] ; then
+    # FIXME: See https://github.com/kubernetes-client/python/issues/866
+    pip install 'kubernetes<10.0.0'
+fi
```

Comment:
Where/how is this script used? I could find no references to it in the repository.

Reply:
It is part of CI builds, used in `.travis.yaml`.